### ISSUE TITLE
Consistently getting console errors from Meta Bind

### ISSUE THREAD

**Issue by ramarivera** [mProjectsCode/obsidian-meta-bind-plugin#210]
### Please fill out these Check-boxes

- [X] I checked that the plugin is up to date
- [X] The issue persist with all other plugins and themes disabled

### This Issue Occurs on

- [X] Windows
- [ ] Linux
- [X] macOS
- [ ] Android
- [ ] iOS

### Plugin Version

0.12.0

### Describe the Issue

When editing or renaming my notes, I noticed that I consistently receive console errors from metabind.
Since I have the console open quite a lot, this is a bit annoying 😅 

```
app.js:1 Uncaught Error: [MB_INTERNAL] "action not permitted" caused by "source 'scope' should have no cache items or subscriptions"
    at hd.getCacheItemForStoragePath (plugin:obsidian-meta-bind-plugin:114:4040)
    at nf.deleteCachesForStoragePath (plugin:obsidian-meta-bind-plugin:57:8395)
    at nf.onStoragePathRenamed (plugin:obsidian-meta-bind-plugin:57:8884)
    at eval (plugin:obsidian-meta-bind-plugin:114:7835)
    at e.tryTrigger (app.js:1:712355)
    at e.trigger (app.js:1:712288)
    at t.trigger (app.js:1:729920)
    at t.onChange (app.js:1:720723)
    at e.trigger (app.js:1:542391)
    at e.<anonymous> (app.js:1:533235)
    at app.js:1:237258
    at Object.next (app.js:1:237363)
    at a (app.js:1:236081)
```

Is there some consideration or configuration I could check in order to fix this?

### Steps to Reproduce

Unfortunately I am not sure how to consistently reproduce this.

### Expected Behavior

I don't expect to see so many unhandled errors on the console

**Comment by mProjectsCode** [comment]
Fixed in 0.12.1

**Comment by ramarivera** [comment]
Cool, thanks!
